python-template/{{cookiecutter.placeholder_repo_name}}/.readthedocs.yml

```diff
@@ -9,7 +9,14 @@ version: 2
 sphinx:
   configuration: docs/conf.py
 
+build:
+  os: "ubuntu-22.04"
+  tools:
+    python: "3.8"
+
 python:
   version: 3.8
   install:
     - requirements: requirements/doc.txt
+    - method: pip
+      path: .
```

Comment:
@dianakhuang: Did you find any information about this, or are you just following what was done in edx-platform?

Reply:
I was just following what was done in edx-platform under the assumption that getting a working RTD config was more important than worrying about whether we may have redundant config.

Reply:
FYI, this is needed in edx-platform because we have to `pip install .` in order to add all the plugin entrypoints that edx-platform needs to properly start up django.  Which we do to generate the module docs now.  We shouldn't generally need this in other repos.